third_party/chromium/media/filters/chunk_demuxer.cc

```diff
@@ -1110,6 +1122,12 @@ base::TimeDelta ChunkDemuxer::GetWriteHead(const std::string& id) const {
   return iter->second[0]->GetWriteHead();
 }
 
+void ChunkDemuxer::SetSourceBufferStreamMemoryLimit(const std::string & id,
```

Comment:
nit: "const std::string & guid" => "const std::string& guid", and the same to other occurrences.